modules/luci-base/po/de/base.po

```diff
@@ -136,7 +136,7 @@ msgstr "802.11w: Schutz von Management-Frames aktivieren"
 
 #: modules/luci-mod-network/luasrc/model/cbi/admin_network/wifi.lua:1130
 msgid "802.11w maximum timeout"
-msgstr "802.11w: Maximales Timeout"
+msgstr "802.11w: Maximale Zeitbegrenzung"
```

Comment:
That sounds weird in german is there not a nice translation? I think timeout is ok or what do you think about `Zeitüberschreitung`?

Reply:
`Zeitüberschreitung` does not feel like the correct translation of `timeout` to me. Other people may have a better feeling for this ...